src/ruby.rs

```diff
@@ -741,4 +742,57 @@ tests! {
             "__test_define_module_function_arity_3__ works!".to_ruby()
         );
     }
+
+    #[test]
+    fn test_undef_method(assert: &mut Assertions) {
+        let class = unsafe {
+            rb_define_class(
+                cstr!("TestUndefClass"),
+                rb_cObject
+            )
+        };
+
+        unsafe { rb_undef_method(class, cstr!("to_s")) };
+
+        assert.rb_eq(
+            lazy_eval("::TestUndefClass.new.respond_to?(:to_s)"),
+            unsafe { Qfalse }
+        );
+    }
+
+       #[test]
+    fn test_undef_module_method(assert: &mut Assertions) {
```

Comment:
Once we have a way to `include` a module from Rust, I think we can do something similar to the above instead